server/dive_server/serializers/viame.py

```diff
@@ -280,17 +280,20 @@ def export_tracks_as_csv(
     filenames=None,
     fps=None,
     header=True,
+    typeFilter=set(),
```

Comment:
Might be an opportunity to add some more dataset export testing to `test_serialize_viame_csv.py`?  I'd check with @subdavis and see what he thinks about possible adding the typeFiltering to here.  Could also be an indicator that we start adding nodejs serializer testing to match the python.  Ensuring that same data in = same data out between the two.

Reply:
Agree with Bryon